lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -706,13 +706,12 @@ static SmallVector<Value> createNdDescriptorTiles(
     Value newRowOffs = rewriter.create<arith::ConstantIndexOp>(loc, i);
     for (int j = 0; j < loadShape[1]; j += descTile[1] * arrayLength) {
       Value newColOffs = rewriter.create<arith::ConstantIndexOp>(loc, j);
-      if (transpose) {
-        std::swap(newRowOffs, newColOffs);
-      }
```

Comment:
placing `std::swap` inside a nested for-loop was a bad idea since it swaps the values each iteration producing non-sense offsets at the end